src/components/PrivacyPolicy.vue

```diff
@@ -0,0 +1,68 @@
+
+        <h2>Links To External Sites</h2>
+        <p>Our Service may contain links to external sites that are not operated by us. If you click on a third party link, you will be directed to that third party's site. We strongly advise you to review the Privacy Policy and terms and conditions of every site you visit.</p>
+        <p>We have no control over, and assume no responsibility for the content, privacy policies or practices of any third party sites, products or services.</p>
+
+        <h2>Aggregated Statistics</h2>
+        <p>Technical University of Denmark may collect statistics about the behavior of visitors to its website. Technical University of Denmark may display this information publicly or provide it to others. However, Technical University of Denmark does not disclose your personally-identifying information.</p>
+
+        <h2>Cookies</h2>
+        <p>To enrich and perfect your online experience, Technical University of Denmark uses "Cookies", similar technologies and services provided by others to display personalized content, appropriate advertising and store your preferences on your computer.</p>
+        <p>A cookie is a string of information that a website stores on a visitor's computer, and that the visitor's browser provides to the website each time the visitor returns. Technical University of Denmark uses cookies to help Technical University of Denmark identify and track visitors, their usage of https://caffeine.dd-decaf.eu, and their website access preferences. Technical University of Denmark visitors who do not wish to have cookies placed on their computers should set their browsers to refuse cookies before using Technical University of Denmark's websites, with the drawback that certain features of Technical University of Denmark's websites may not function properly without the aid of cookies.</p>
+        <p>By continuing to navigate our website without changing your cookie settings, you hereby acknowledge and agree to Technical University of Denmark's use of cookies.</p>
+
+        <h2>Privacy Policy Changes</h2>
+        <p>Although most changes are likely to be minor, Technical University of Denmark may change its Privacy Policy from time to time, and in Technical University of Denmark's sole discretion. Technical University of Denmark encourages visitors to frequently check this page for any changes to its Privacy Policy. Your continued use of this site after any change in this Privacy Policy will constitute your acceptance of such change.</p>
+
+        <h2></h2>
+        <p></p>
```

Comment:
These can be removed

Reply:
> I do like kebab-case in the URLs, but it's inconsistent with `/interactiveMap`. Should we change that to `/interactive-map` then?

Yes. All routes should be overhauled. `/pathways` should be `design` etc.